contrib/test/integration/main.yml

```diff
@@ -30,7 +30,7 @@
 
     - name: clone build and install runc
       include: "build/runc.yml"
-      when: "{{ not use_system_runc | default(True) | bool}}"
+      when: "{{ not use_system_runc | default(False) | bool}}"
```

Comment:
can you also change the "not use_system_runc" to "use_system_runc" ?

Reply:
Nope, that brings us back to the bug :) we don't want to skip runc build by default, we want to build runc **always**

Reply:
I can do:

```
when: "{{ don_t_use_system_runc | default(True) | bool}}"
```
if it's more readable to you

Reply:
@giuseppe Does the default apply to the task or the use_system_runc variable?

Reply:
it applies to the whole expression preceding the pipe once evaluated

Reply:
I think the problem is that [the `default` filter][1] is only switching on whether its input is `None` or not.  So you could have:

    {{ use_system_runc | default(False) | not | bool }}

except that there seems to be no `not` filter.  I recommend we change this to:

    {{ build_runc | default(True) | boolean }}

and @guiseppe can set `build_runc` false when he wants to use the system `runc`.

[1]: http://docs.ansible.com/ansible/latest/playbooks_filters.html#defaulting-undefined-variables

Reply:
> {{ build_runc | default(True) | boolean }}

AMIs are almost built to unblock #1297 - I'll change the filter to the one you suggested in a follow up!
Thanks!